packages/builders/scripts/orchestration/helpers.js

```diff
@@ -17,7 +17,8 @@
  */
 export const parseChainHubOpts = async scriptArgs => {
   // import dynamically so the modules can work in CoreEval environment
-  const { parseArgs } = await import('node:util');
+  const avoidBundling = 'node:util';
```

Comment:
I suppose esbuild it bundling `node-util` when it sees the literal. But the code here shouldn't be concerned with that.

Instead please configure esbuild with the `--external` option to exclude what shouldn't be bundled.

Reply:
Oops, now I hit a different hurdle that makes `--external` fail for a different reason.  We need to package the esbuild output with the original `@endo/bundle-source` implementation, and that chokes on all the modules that esbuild has passed through.

It seems `@endo/compartment-mapper` hasn't sufficiently implemented `externals` for this use, but I'll ask @kriskowal if there's a way to make this work.